scripts/unittests/run_all_cpp_test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+BUILD_DIR="build"
+
+if [ ! -d $BUILD_DIR ]; then
+    echo "This script should be run from the root of the project."
+    exit 0
+fi
+
+cd $BUILD_DIR
+# use ctest -R name_of_test to run a specific test
+ctest
```

Comment:
I'm not sure if this is the best way to run all of the C++ tests. I've used the simplest method to run all installed C++ unit tests, which requires the project to be built with `WITH_TEST` enabled.